README.md

```diff
@@ -543,6 +544,31 @@ This will be rendered using a select box that way:
 
 Note that string representations of numbers will be cast back and reflected as actual numbers into form state.
 
+### Disabled attribute for `enum` fields
+
+This library supports the 'disbaled' property for `enum` fields, which allows disabling 'enum' fields.This feature is a part of uiSchema.
```

Comment:
`"disabled"`. Space after period. Actually, I find this sentence a bit confusing. Maybe change "disabling 'enum' fields" to "disabling some options of an `enum`".